src/ui/QuerySuggestPreview/QuerySuggestPreview.ts

```diff
@@ -186,7 +186,6 @@ export class QuerySuggestPreview extends Component implements IComponentBindings
 
   private handleSelect(suggestionText: string, element: HTMLElement, rank: number) {
     this.logClickQuerySuggestPreview(suggestionText, rank, element);
-    element.click();
```

Comment:
Echoing Lucille's comment, could we have a test checking that we are logging to analytics only one time when clicking a preview?

Reply:
Good point, I'll add a test for that.